telegram/ext/_updater.py

```diff
@@ -122,6 +122,12 @@ def __init__(
         self.__lock = asyncio.Lock()
         self.__polling_task: Optional[asyncio.Task] = None
 
+    def __repr__(self) -> str:
+        return (
+            f"Updater for bot {self.bot}. Number of updates currently in the queue: "
```

Comment:
I vote to leave out the queue status - argumentation see `Job.__repr__`

Reply:
Done